nahid/xWeather/Program.cs

```diff
@@ -10,7 +10,8 @@ class Program
     {
         static void Main(string[] args)
         {
-            IInputHandler inputHandler = InputHandlerFactory.CreateInputHandler(args);
```

Comment:
Please don't remove existing code. Use the same handler.

Reply:
I have updated the program.cs